docker/uwsgi.ini

```diff
@@ -0,0 +1,45 @@
+[uwsgi]
+; The IP address (typically localhost) and port that the WSGI process should listen on
+http = 0.0.0.0:8080
+https = 0.0.0.0:8443,/opt/nautobot/nautobot.crt,/opt/nautobot/nautobot.key
+
+; Fail to start if any parameter in the configuration file isn’t explicitly understood by uWSGI
+strict = true
+
+; Enable master process to gracefully re-spawn and pre-fork workers
+master = true
+
+; Allow Python app-generated threads to run
+enable-threads = true
+
+;Try to remove all of the generated file/sockets during shutdown
+vacuum = true
+
+; Do not use multiple interpreters, allowing only Nautobot to run
+single-interpreter = true
+
+; Shutdown when receiving SIGTERM (default is respawn)
+die-on-term = true
+
+; Prevents uWSGI from starting if it is unable load Nautobot (usually due to errors)
+need-app = true
+
+; By default, uWSGI has rather verbose logging that can be noisy
+disable-logging = true
+
+; Assert that critical 4xx and 5xx errors are still logged
+log-4xx = true
+log-5xx = true
+
+;
+; Advanced settings (disabled by default)
+; Customize these for your environment if and only if you need them.
+; Ref: https://uwsgi-docs.readthedocs.io/en/latest/Options.html
+;
+
+; Number of uWSGI workers to spawn. This should typically be 2n+1, where n is the number of CPU cores present.
+; processes = 5
```

Comment:
feels like it should be at least 3 by default because `n` is never gonna be smaller than 1